README.md

```diff
@@ -1,4 +1,4 @@
-# Duke project template
+# duke.Duke project template
```

Comment:
I love how when you refactored the duke class, you updated the readme files accordingly

Reply:
Thanks!